coral/src/app/layout/Header.tsx

```diff
@@ -0,0 +1,96 @@
+import { Flexbox, GridItem, Icon, Tooltip } from "@aivenio/design-system";
+import questionMark from "@aivenio/design-system/dist/module/icons/questionMark";
+import user from "@aivenio/design-system/dist/module/icons/user";
+import notifications from "@aivenio/design-system/dist/module/icons/notifications";
+import data from "@aivenio/design-system/dist/src/icons/console";
+
+const originLocationKlawAngular = window.location.origin;
+
+function createHeaderMenuListItem({
```

Comment:
I thought about that, but then we have a component with basically being a`li` element, which we avoided for `TopicCards`. We could use the `a` as first element in the utility component, tho 🤔  

I mostly added it there instead of a separate component to avoid duplicated tests in `Header` and `HeaderLinks` 😅 But I can change that! At some point we will have more functionality there and will have to separate it anyway!